README.md

```diff
@@ -64,42 +64,43 @@ id="anchor-versions">[1](#footnote-versions)</sup>
 
 Log a message and 10 fields:
 
-| Package             |    Time     | Time % to zap | Objects Allocated |
-| :------------------ | :---------: | :-----------: | :---------------: |
-| zerolog             |   389 ns/op |     -53%      |    1 allocs/op    |
-| :zap: zap           |   830 ns/op |      +0%      |    5 allocs/op    |
-| :zap: zap (sugared) |  1377 ns/op |     +66%      |   10 allocs/op    |
-| go-kit              |  4311 ns/op |     +419%     |   57 allocs/op    |
-| apex/log            | 29235 ns/op |    +3422%     |   63 allocs/op    |
-| logrus              | 31084 ns/op |    +3645%     |   79 allocs/op    |
-| log15               | 31085 ns/op |    +3645%     |   74 allocs/op    |
+| Package | Time | Time % to zap | Objects Allocated |
+| :------ | :--: | :-----------: | :---------------: |
+| :zap: zap | 2012 ns/op | +0% | 5 allocs/op
+| :zap: zap (sugared) | 2500 ns/op | +24% | 10 allocs/op
+| zerolog | 1254 ns/op | -38% | 1 allocs/op
+| slog | 5711 ns/op | +184% | 40 allocs/op
+| go-kit | 6329 ns/op | +215% | 57 allocs/op
+| apex/log | 19714 ns/op | +880% | 63 allocs/op
+| log15 | 25503 ns/op | +1168% | 74 allocs/op
+| logrus | 25816 ns/op | +1183% | 79 allocs/op
 
 Log a message with a logger that already has 10 fields of context:
 
-| Package             |    Time     | Time % to zap | Objects Allocated |
-| :------------------ | :---------: | :-----------: | :---------------: |
-| zerolog             |    32 ns/op |     -27%      |    0 allocs/op    |
-| :zap: zap           |    44 ns/op |      +0%      |    0 allocs/op    |
-| :zap: zap (sugared) |    74 ns/op |     +68%      |    1 allocs/op    |
-| go-kit              |  5072 ns/op |   +11427%     |   56 allocs/op    |
-| log15               | 22634 ns/op |   +51341%     |   70 allocs/op    |
-| apex/log            | 28775 ns/op |   +65298%     |   53 allocs/op    |
-| logrus              | 29047 ns/op |   +65916%     |   68 allocs/op    |
+| Package | Time | Time % to zap | Objects Allocated |
+| :------ | :--: | :-----------: | :---------------: |
+| :zap: zap (sugared) | 265 ns/op | -31% | 1 allocs/op
+| :zap: zap | 386 ns/op | +0% | 0 allocs/op
```

Comment:
This one looks wrong?

Reply:
As in, sugared is somehow faster than non-sugared.

Also, looks like previously someone (manually?) sorted zerolog to be on top?

Reply:
Huh, yeah that's weird. I haven't had a chance to dig into it yet.

Yeah, I think when this was last updated, the author aligned the table and sorted them by time rather than "zap on top and then everything else sorted" which is what the script does.

Reply:
hmm i re-generated the readme again a bunch of times and now i'm consistently getting the expected result (i.e. sugared logger is slower by ~20-30%)

i suspect that i likely unintentionally strained my cpu doing something else on my laptop while the sugared benchmark was running (sorry! noob in benchmarking)